src/main/java/com/training/utilities/RequestBody.java

```diff
@@ -0,0 +1,20 @@
+package com.training.utilities;
+
+public class RequestBody {
+
+   public String getGuestAccountInvalidEmailBody(){
+        return "{\"email\": \"testPranav@@api.com\"}";
+    }
+
+    public String getGuestAccountWrongEmailBody(){
+       return "{\"email\": \"assignment70test@api.com\"}";
+    }
+
+    public String getGuestAccountWrongAppKeyBody(){
+       return "{\"email\": \"testPranav@api.com\"}";
+    }
+
+    public String getGuestAccountValidationBody(){
+       return "{\"email\": \"testPranav@api.com\"}";
```

Comment:
Incorrect design : You should be following the approach which we used for Response pojos. Variables & getters. In this will be use setters.
And we will be using these setters in our test class to set values and then pass it to RestEngine.